### ISSUE TITLE
Have a QA site

### ISSUE THREAD

**Issue by rhoegg** [techlahoma/techlahoma_donations#57]

**Comment by jagthedrummer** [comment]
@rhoegg I'm guessing that you have something in mind that's different from our current staging site? https://techlahoma-donations-staging.herokuapp.com/

**Comment by rhoegg** [comment]
Maybe not, let me look at it and see if there's any value I can add

**Comment by jagthedrummer** [comment]
The CI/auto-deployment stuff that you mentioned in Slack earlier would be awesome.

**Comment by rhoegg** [comment]
Yeah makes sense. Can you send me what you do now to deploy staging and production, and how you make sure things are cool afterward?

**Comment by jagthedrummer** [comment]
@rhoegg It's currently a manual push to heroku for either. I have the staging site setup as a git remote called `staging` and the production site set up as a git remote called `production`.

To test a branch on the staging site I do something like:

```
git push staging my-branch:master
```

When I'm happy with it, I merge that branch to master, and then push master to production.

```
git push production master
```

On both sites to make sure things are cool, I load the site, click around, and make a (sometimes small) donation.

**Comment by rhoegg** [comment]
Right, I think I need access to the same heroku account you're using,
right? Where do you set up the DNS?

**Comment by jagthedrummer** [comment]
@rhoegg So sorry for the delay. I somehow missed this when you last responded. If you'll DM me your heroku email address I'll invite you to both apps. I wanna say it's DNSimple where we have the DNS setup. I think @seejessicacode may be able to help us with that.

**Comment by seejessicacode** [comment]
donate.techlahoma.org points to donate.techlahoma.org.herokudns.com

**Comment by jagthedrummer** [comment]
@seejessicacode where do we manage those records?

**Comment by seejessicacode** [comment]
DNSimple.com, I've just shared management permissions with your Techlahoma gmail.

**Comment by jagthedrummer** [comment]
@rhoegg, this looks interesting : https://devcenter.heroku.com/articles/heroku-ci
